src/main/java/com/cleanroommc/groovyscript/sandbox/GroovyScriptSandbox.java

```diff
@@ -181,6 +192,7 @@ public <T> T runClosure(Closure<T> closure, Object... args) {
                 return new AtomicInteger();
             }).addAndGet(1);
         } finally {
+            securityManager.uninstall();
```

Comment:
Should be fine for now, not sure if there are any drastic performance penalties.